CentralComputing/Pod.cpp

```diff
@@ -1,33 +1,61 @@
 #include "Sensor_Package.h"
+#include "Pod.h"
 
 #include <thread>
+#include <csignal>
 
 using namespace std;
 
 Sensor_Package * sensors;
+bool running = true;
```

Comment:
This should be declared `volatile` since it is used in a signal handler. `volatile bool running = true;`